<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SCIENTISTS &amp; COMPUTERS</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tbFind">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>580</width>
      <height>561</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="documentMode">
     <bool>true</bool>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tbScientist">
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <attribute name="title">
      <string>Scientists</string>
     </attribute>
     <widget class="QLabel" name="lblDodS">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Died:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblNameS">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>53</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblDobS">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Born:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="edtNameS">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>40</y>
        <width>251</width>
        <height>22</height>
       </rect>
      </property>
      <property name="inputMask">
       <string/>
      </property>
     </widget>
     <widget class="QColumnView" name="colScientists">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>561</width>
        <height>351</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pbAddS">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>130</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="grbSortS">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>10</y>
        <width>201</width>
        <height>161</height>
       </rect>
      </property>
      <property name="title">
       <string>Sort by</string>
      </property>
      <widget class="QComboBox" name="cmbSortS">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>181</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Name</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Born</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Died</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Gender</string>
        </property>
       </item>
      </widget>
      <widget class="QRadioButton" name="rbAscS">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>95</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Ascending</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="rbDescS">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>80</y>
         <width>95</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Descending</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pbSortS">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>120</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Sort</string>
       </property>
      </widget>
     </widget>
     <widget class="QLineEdit" name="edtDobS">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>70</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="inputMask">
       <string extracomment="    -  -  "/>
      </property>
     </widget>
     <widget class="QLineEdit" name="edtDodS">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>100</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="inputMask">
       <string/>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>351</width>
        <height>161</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Add a new Scientist</string>
      </property>
      <widget class="QRadioButton" name="rbFemaleS">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>60</y>
         <width>70</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Female</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="rbMaleS">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>60</y>
         <width>60</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Male</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rbBothS">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>90</y>
         <width>70</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Both</string>
       </property>
      </widget>
     </widget>
     <zorder>groupBox</zorder>
     <zorder>lblDodS</zorder>
     <zorder>lblNameS</zorder>
     <zorder>lblDobS</zorder>
     <zorder>edtNameS</zorder>
     <zorder>colScientists</zorder>
     <zorder>pbAddS</zorder>
     <zorder>grbSortS</zorder>
     <zorder>edtDobS</zorder>
     <zorder>edtDodS</zorder>
    </widget>
    <widget class="QWidget" name="tbComputers_2">
     <attribute name="title">
      <string>Computers</string>
     </attribute>
     <widget class="QLabel" name="lblBrand">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>53</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Brand:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="edtBrand">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>40</y>
        <width>251</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lblType">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>53</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Type:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="edtYear">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>100</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="edtType">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>70</y>
        <width>251</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_2">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>100</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Built</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="lblYear">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>53</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Year:</string>
      </property>
     </widget>
     <widget class="QColumnView" name="colComputers">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>561</width>
        <height>351</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pbAddC">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>130</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="grpSortC">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>10</y>
        <width>201</width>
        <height>161</height>
       </rect>
      </property>
      <property name="title">
       <string>Sort by</string>
      </property>
      <widget class="QComboBox" name="cmbComputers">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>181</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Brand</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Type</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Year</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Built</string>
        </property>
       </item>
      </widget>
      <widget class="QRadioButton" name="rbAscC">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>95</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Ascending</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="rbDescC">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>80</y>
         <width>95</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Descending</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pbSortC">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>120</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Sort</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>351</width>
        <height>161</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Add a new Computer</string>
      </property>
     </widget>
     <zorder>groupBox_3</zorder>
     <zorder>lblBrand</zorder>
     <zorder>edtBrand</zorder>
     <zorder>lblType</zorder>
     <zorder>edtYear</zorder>
     <zorder>edtType</zorder>
     <zorder>checkBox_2</zorder>
     <zorder>lblYear</zorder>
     <zorder>colComputers</zorder>
     <zorder>pbAddC</zorder>
     <zorder>grpSortC</zorder>
    </widget>
    <widget class="QWidget" name="tbConnect">
     <attribute name="title">
      <string>Connect</string>
     </attribute>
     <widget class="QLabel" name="lblConnectSwithC">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>300</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Connect a Scientist with a Computer</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblInputS">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>130</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Input the Scientist ID:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>60</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lblInputC">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>130</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Input the Computer ID:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>100</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>100</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblConnectComputerScientist">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>300</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Connect a Computer with a Scientist</string>
      </property>
     </widget>
     <widget class="QLabel" name="InputComputer2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>130</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Input the Computer ID:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>280</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>280</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_5">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>240</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="InputScientist2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>280</y>
        <width>130</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Input the Scientist ID:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblComputern_no1">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>150</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>with Computer no: </string>
      </property>
     </widget>
     <widget class="QLabel" name="lblScientist_connect">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>170</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="lblScientistComputer">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>221</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>You have now connected Scientist no: </string>
      </property>
     </widget>
     <widget class="QLabel" name="lblComputer_no1">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>150</y>
        <width>30</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="lblScientist_no2">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>330</y>
        <width>30</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="lblComputer_no2">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>330</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>with Computer no: </string>
      </property>
     </widget>
     <widget class="QLabel" name="lblComputerScientist">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>330</y>
        <width>221</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>You have now connected Scientist no: </string>
      </property>
     </widget>
     <widget class="QLabel" name="lblScientist_no1">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>150</y>
        <width>30</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Find</string>
     </attribute>
     <widget class="QColumnView" name="colFind">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>561</width>
        <height>351</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>80</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Search term:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>70</y>
        <width>300</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>561</width>
        <height>141</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="lblHeader">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>561</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Find a Scientist or a Computer</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>100</y>
         <width>95</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Scientist</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_2">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>100</y>
         <width>95</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Computer</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>100</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </widget>
     <zorder>groupBox_2</zorder>
     <zorder>colFind</zorder>
     <zorder>label</zorder>
     <zorder>lineEdit</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>620</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuNew">
     <property name="title">
      <string>New</string>
     </property>
     <addaction name="actionScientist"/>
     <addaction name="actionComputer"/>
     <addaction name="actionConnection"/>
    </widget>
    <widget class="QMenu" name="menuFind">
     <property name="title">
      <string>Find</string>
     </property>
     <widget class="QMenu" name="menuScientist">
      <property name="title">
       <string>Scientist</string>
      </property>
      <addaction name="actionBy_name"/>
     </widget>
     <addaction name="menuScientist"/>
     <addaction name="actionComputer_2"/>
    </widget>
    <addaction name="menuNew"/>
    <addaction name="menuFind"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionScientist">
   <property name="text">
    <string>Scientist</string>
   </property>
  </action>
  <action name="actionComputer">
   <property name="text">
    <string>Computer</string>
   </property>
  </action>
  <action name="actionConnection">
   <property name="text">
    <string>Connection</string>
   </property>
  </action>
  <action name="actionComputer_2">
   <property name="text">
    <string>Computer</string>
   </property>
  </action>
  <action name="actionConnection_2">
   <property name="text">
    <string>Connection</string>
   </property>
  </action>
  <action name="actionBy_name">
   <property name="text">
    <string>By name</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>tbFind</tabstop>
  <tabstop>edtNameS</tabstop>
  <tabstop>edtDobS</tabstop>
  <tabstop>edtDodS</tabstop>
  <tabstop>rbFemaleS</tabstop>
  <tabstop>rbMaleS</tabstop>
  <tabstop>rbBothS</tabstop>
  <tabstop>pbAddS</tabstop>
  <tabstop>cmbSortS</tabstop>
  <tabstop>rbAscS</tabstop>
  <tabstop>rbDescS</tabstop>
  <tabstop>pbSortS</tabstop>
  <tabstop>colScientists</tabstop>
  <tabstop>edtBrand</tabstop>
  <tabstop>edtType</tabstop>
  <tabstop>edtYear</tabstop>
  <tabstop>checkBox_2</tabstop>
  <tabstop>pbAddC</tabstop>
  <tabstop>cmbComputers</tabstop>
  <tabstop>rbAscC</tabstop>
  <tabstop>rbDescC</tabstop>
  <tabstop>colComputers</tabstop>
  <tabstop>pbSortC</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
